app/views/ChangeAvatarView/index.tsx

```diff
@@ -130,18 +130,19 @@ const ChangeAvatarView = () => {
 		return navigation.goBack();
 	};
 
-	const pickImage = async () => {
+	const pickImage = async (isCam: boolean) => {
 		const options = {
 			cropping: true,
 			compressImageQuality: 0.8,
 			freeStyleCropEnabled: true,
 			cropperAvoidEmptySpaceAroundImage: false,
 			cropperChooseText: I18n.t('Choose'),
 			cropperCancelText: I18n.t('Cancel'),
-			includeBase64: true
+			includeBase64: true,
+			useFrontCamera: true
```

Comment:
```suggestion
			useFrontCamera: isCam
```

Reply:
Done with the changes. thanks for the valuable insights and feedback. please let me know if any further improvements are needed.